### ISSUE TITLE
Bot users don’t render quite right

### ISSUE THREAD

**Issue by ticky** [release-drafter/release-drafter#60]
It seems that bot users are inlined as usual, and count as contributors, which results in a bit of unusual rendering:

>* 7: Update dependency emoji-regex to v7.0.1 by @renovate[bot]
>
>Thanks to @renovate-bot, @renovate[bot] and @ticky!

**Comment by toolmantim** [comment]
Yeah, that seems to be a general problem  across GitHub right now. I was assuming GitHub would update their Markdown renderer to point them to the app page for the bot user, at some point. But we can do better in the meantime! Any suggestions?

For a start, we should filter bots out of `$CONTRIBUTORS`

**Comment by ticky** [comment]
Yeah, I’d suggest filtering them from contributors.

As far as for people concatenating it with an @, I’d suggest making the @ part of the variable, but in the interests of backward-compatibility, maybe it should prefix bots with a zero-width joiner? e.g. @&zwj;renovate[bot]

**Comment by jamietanna** [comment]
I've taken a stab at this with https://github.com/release-drafter/release-drafter/pull/1376 - happy to add in the zero-width joiner too / instead of Markdown linking to the app page?
